### DISCUSSION TITLE
fixing-the-error-program-has-more-than-one-entry-point-defined-for-console-apps-containing-xunit-tests/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#40] [Announcements]
# fixing-the-error-program-has-more-than-one-entry-point-defined-for-console-apps-containing-xunit-tests/

In this post I'll describe how to fix the error "Program has more than one entry point defined" when you try and make a .NET Core test project a console app

https://andrewlock.net/fixing-the-error-program-has-more-than-one-entry-point-defined-for-console-apps-containing-xunit-tests/

**Comment by RegencySoftware** [comment]
Been using Rider for sometime now, and a new project in VisualStudio started to hit this issue. Great blog post, very helpful!

[a comment by an automated account is not shown]

**Comment by caccorsi** [comment]
Thank you for TL;DR; !
